product_planned_price/models/product_template.py

```diff
@@ -24,11 +24,29 @@ class ProductTemplate(models.Model):
         'other parameters.',
     )
     list_price_type = fields.Selection([
-        ('manual', 'Fixed value')],
+        ('manual', 'Fixed value'),
+        ('by_margin', 'By Margin')
+    ],
         string='Planned Price Type',
         # we make it optional
         # required=True,
         default='manual',
+        help="If the 'manual' type is chosen, a fixed planned"
```

Comment:
Hi Nico! If you see, the 'manual' type is rendered by 'Fixed value'. May you can change in the help 'manual' for 'Fixed value'.
![seleccion_008](https://user-images.githubusercontent.com/29925834/42043820-bb6ffd40-7acd-11e8-8985-5ec459a8cb9e.png)

Reply:
Thanks Valentino!
I've fix it now